rust/arrow/src/array/data.rs

```diff
@@ -128,6 +128,11 @@ impl ArrayData {
         &self.null_bitmap
     }
 
+    /// Returns a reference to the null buffer of this array data.
+    pub fn null_buffer(&self) -> Option<&Buffer> {
+        self.null_bitmap().as_ref().map(|b| b.buffer_ref())
+    }
+
```

Comment:
Thanks @liurenjie1024, I know it's a small addition, but can you add an unit test so we don't drop our coverage % (even though we have stopped tracking it for now).